css/main.scss

```diff
@@ -1,50 +1,54 @@
 ---
-
```

Comment:
This file seems to have mostly code style changes unrelated to the "Code of Conduct" PR topic. I guess it's fine but I would have made these changes outside of this PR for clarity.

Reply:
Prettier being prettier. Basically the only change on that file is at the bottom where I added some styling and text-align for code of conduct page.